infra/terraform/test-org/org/gsuite.tf

```diff
@@ -99,8 +103,61 @@ resource "google_billing_account_iam_member" "ci_gsuite_sa_billing" {
   member             = "serviceAccount:${google_service_account.ci_gsuite_sa.email}"
 }
 
-# Grant G-Suite project rights to cft_ci_group
-# Required to be able to create keys for the gsuite sa.
+// Generate a json key and put it into the secrets bucket.
+
+resource "google_service_account_key" "ci_gsuite_sa" {
+  service_account_id = google_service_account.ci_gsuite_sa.id
+}
+
+provider "google" {
```

Comment:
I don't see any need for a separate provider config.

Reply:
Yep, thanks. I was testing it locally launching from the service account, trying to store and retrieve it, and mixed it up. We have to use a custom provider config launched from SA and not personal, to retrieve the key, since signed url can be created only under SA.

Fixed and updated the retrieval example in the PR description.